### ISSUE TITLE
[Catalog] Link not taking users to separate page

### ISSUE THREAD

**Issue by theBeginner86** [meshery/meshery.io#1323]
### Description

Currently, the link on WASM filter is not taking users to a separate dedicated page like designs.

https://github.com/meshery/meshery.io/assets/73700530/82932ecf-f61d-4109-94d9-3d72b9c5a16a

### Expected Behavior

Ensure that users are taken to a separate page when they click on the link at the top right corner of the modal

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

**Comment by Adi9235** [comment]
Hey @theBeginner86 can I work on this issue ?

**Comment by nakulbh** [comment]
hey @theBeginner86 can i work on this issue

**Comment by theBeginner86** [comment]
Sure, @Adi9235 go ahead 👍

**Comment by theBeginner86** [comment]
@nakulbh you may please take up some other issue. 
https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+user%3Alayer5io+user%3Ameshery+user%3Aservice-mesh-performance+no%3Aassignee+

**Comment by Ghat0tkach** [comment]
@Adi9235  , the website just got update and we have commented out the model box pop-up , however the issue still persist through the individual cards.

![image](https://github.com/meshery/meshery.io/assets/59855919/7f89f378-a5e6-44c5-8a0d-327a651c8282)

Still , You would have to make the redirection of these cards  on click

**Comment by sarkartanmay393** [comment]
@Ghat0tkach Where should the wasm cards redirect to? Other cards here redirects to a page, but for that data is coming from `/collections/_catalog/`, and the wasm cards are getting data from `/_data` directory. Isn't it complex for a goodfirstissue? Or am I missing anything?

**Comment by abhijeetgauravm** [comment]
@theBeginner86 @Ghat0tkach where the wasm filter should redirect?

**Comment by Adi9235** [comment]
@theBeginner86 @Ghat0tkach I was also trying to figure out the link to connect to wasm filter can you please help out where to find the information regarding that.

**Comment by thisiskaransgit** [comment]
@Adi9235, I'll take over this issue.

**Comment by sagniksantra** [comment]
@theBeginner86 can you please assign this issue to me... I can work on it.

**Comment by theBeginner86** [comment]
> @theBeginner86 can you please assign this issue to me... I can work on it.

sounds good!

**Comment by iArchitSharma** [comment]
@sagniksantra are you still working on this issue?

**Comment by sagniksantra** [comment]
No...  I wasn't able to understand how to proceed with the issue.

**Comment by iArchitSharma** [comment]
Okay, no worries. I'll take on this issue. Meanwhile, there are some other issues that you can take on.

**Comment by sagniksantra** [comment]
Sure. Thanks

**Comment by Lucif3r-in** [comment]
Sure @iArchitSharma will look into it but I am facing some issues while starting the server
![image](https://github.com/meshery/meshery.io/assets/85403534/17813423-ce90-45f0-a796-a6252fdf9014)
Can you please help?
